Cue/app/api/Library.js

```diff
@@ -21,6 +21,22 @@ var LibraryApi = {
 			throw (e)
 		});
 	},
+
+	fetchDeck(uuid : string, state) {
+		let url = serverURL + '/api/v1/deck/' + uuid;
+		let user = state.user;
+		return fetch(url, {
```

Comment:
When we have some time, we should consider refactoring the existing API classes to use a common base method which just takes in the API route and method, handles authentication and error checking, and returns a `Promise`. I feel like that would save us a lot of redundant code.